spec/data_structures_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../two/data_structures.rb'
+
+describe 'data_structures' do
+  it "array cero will be cat" do
```

Comment:
Please change doble quotes  "" to single ones ''

Reply:
done